src/openfermion/utils/_trotter_exp_to_qgates.py

```diff
@@ -220,8 +228,17 @@ def pauli_exp_to_qasm(qubit_operator_list,
             ret_list = ret_list + cnots1
 
             # 3. Rotation (Note kexp & Ntrot)
-            ret_list = ret_list + ["Rz {} {}".format(
-                term_coeff * evolution_time, qids[-1])]
+            if ancilla is not None:
+                if len(qids) > 0:
+                    ret_list = ret_list + ["C-Phase {} {} {}".format(
+                        term_coeff * evolution_time, qids[-1], ancilla)]
+                else:
```

Comment:
I think these swap the more typical convention of control, target vs target, control

Reply:
Fair point, my mistake. It's not an issue for a C-Phase gate as it's symmetric, but we should definitely try to keep a convention.